dstyle.dd

```diff
@@ -135,7 +135,22 @@ ubyte asciiChar;
 -------------------------------
     )
 
+    $(DT $(LNAME2 naming_udas, User-Defined Attributes))
+    $(DD For symbols that are only to be used as user-defined attributes the
+         names should be camelCased, so their first letter is lowercase. This
+         convention takes precedence over any previously mentioned conventions.
+         This matches conventions of the built in attributes like `@safe`,
+         `@nogc` and the special compiler recognized UDA `@selector`.
```

Comment:
Or really, `@noGC`, since `GC` is an acronym, not a word. :)

But it's not like this is the first time that we've been inconsistent with naming, and technically, `@nogc` isn't a UDA.